src/openrct2/windows/EditorObjectSelection.cpp

```diff
@@ -1877,3 +1877,47 @@ bool editor_check_object_group_at_least_one_selected(sint32 checkObjectType)
     }
     return false;
 }
+
+sint32 editor_remove_unused_objects()
+{
+    bool selectionFlagsExist = _objectSelectionFlags != nullptr;
+    if (!selectionFlagsExist)
```

Comment:
Just fold these two `if`s together.

Reply:
Unfortunately, I can't. I changed the name now though. Should be clearer why it is necessary now.

Reply:
You can:
```
if (createSelectionFlags && !sub_6AB211())
{
    return 0;
}
```

Reply:
Overlooked and misunderstood that. Sorry about that.